src/Jackett.Common/Indexers/Abstract/GazelleTracker.cs

```diff
@@ -196,8 +222,12 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
             }
 
             searchUrl += "?" + queryCollection.GetQueryString();
+            var headers = new Dictionary<string, string>();
+            var apiKey = (ConfigurationData.StringItem)configData.GetDynamic("apikey");
+            if (apiKey != null)
+                headers["Authorization"] = apiKey.Value;
```

Comment:
I'm not sure if empty dictionary is the same as null (maybe empty dictionary overwrites default cookies I don't know). 
You can try:
```
var headers = apiKey != null ? new Dictionary<string, string> { "Authorization": apiKey.Value} : null;
```